src/room/InCallView.tsx

```diff
@@ -217,24 +241,27 @@ export function InCallView({
     []
   );
 
-  const renderContent = useCallback((): JSX.Element => {
+  const renderContent = (): JSX.Element => {
```

Comment:
Why not `useCallback()`?

(/me sees the realization that I lack react hooks knowledge coming :sweat_smile:)

Reply:
It's unnecessary since `renderContent` is called directly within this component rather than being passed as a prop, so `useCallback` won't make this any more efficient (it'll just cause React to do additional unnecessary comparisons of the dependencies)

And sorry, I think I was the one to suggest that you put `useCallback` here, I guess I wasn't paying attention to where the function was used at the time

Reply:
> It's unnecessary since `renderContent` is called directly within this component rather than being passed as a prop, so `useCallback` won't make this any more efficient (it'll just cause React to do additional unnecessary comparisons of the dependencies)

Ah, I see! Btw, thank you for the occasional lesson on react-hooks :D (I should probably allocate some time to do some more in-depth research :sweat_smile: )

> And sorry, I think I was the one to suggest that you put `useCallback` here, I guess I wasn't paying attention to where the function was used at the time

No worries!